docs/index.md

```diff
@@ -270,6 +323,7 @@ connection "config" {
 
   ini_paths  = [ "*.ini", "~/*.ini", "/path/to/dir/main.ini" ]
   json_paths = [ "*.json", "~/*.json", "/path/to/dir/main.json" ]
+  toml_paths = [ "*.toml", "/path/to/dir/main.toml" ]
```

Comment:
```suggestion
  toml_paths = [ "*.toml", "~/*.toml", "/path/to/dir/main.toml" ]
```

Could we consider the above suggestions to maintain consistency?

Reply:
> Could we consider the above suggestions to maintain consistency?

Sure, thanks for spotting that!